### DISCUSSION TITLE
支付成功 -- 需要激活码

### DISCUSSION THREAD

**Discussion by YayuZhou123** [getlantern/lantern-forum-cn#589] [General]
![支付成功](https://github.com/getlantern/lantern-forum-cn/assets/172757658/9bd89291-a6a9-4252-9b85-f09604838817)
你好，微信支付成功。
麻烦发激活码给我们。
邮箱：330266506@qq.com
谢谢

**Comment by MaxMKMax** [comment]
您好. 已经帮您提交问题了. 但负责人要周二才回来. 我会帮您追问的. 不好意思让您久等了. 请您过 2-3 天再查看这个帖子或邮箱. 我们会给您发送激活码. 谢谢您支持蓝灯!

**Comment by YayuZhou123** [comment]
@Mike777Khoo 
你好。
麻烦你们弄清楚，@luojw-dwr -- 是你们的另外一个客户，跟我是没有任何关系的。我不认识的。
我的邮箱： 330266506@qq.com -- 没有收到任何激活码。
你们是不是把我支付的费用，合并到他那边去了？
麻烦你们尽快处理一下。
谢谢。

**Reply by MaxMKMax** [reply]
@YayuZhou123 
您好. 好的. 我们清楚是另一个用户. 没有搞错的. 请您放心. 已经帮您提交问题上去了.

谢谢您支持蓝灯. 过几天请查看这个帖子或邮箱. 谢谢!

**Comment by MaxMKMax** [comment]
@YayuZhou123 
您好. 这是您专属的激活码:
QDD8M-KBVDQ-G4FMT-W6YXB-JKRQB

**Comment by YayuZhou123** [comment]
你好，


你看我手机上的是最新版本蓝灯吧。
可是输入激活码时显示试图购买时出错。
请问激活码
QDD8M-KBVDQ-G4FMT-W6YXB-JKRQB
对应的邮箱是： ***@***.***


这个正确吗？
登录不上去呀。




期待回复。
谢谢。





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月19日(星期三) 凌晨2:54
***@***.***&gt;;
抄送:&nbsp;"Ran zhou ***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 支付成功 -- 需要激活码 (Discussion #589)





 
@YayuZhou123
 您好. 这是您专属的激活码:
 QDD8M-KBVDQ-G4FMT-W6YXB-JKRQB

**Comment by YayuZhou123** [comment]
你好，


尽快帮我处理呀。真的很着急使用。
激活码 +&nbsp;邮箱 --&nbsp;还是登录不了。


谢谢。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月19日(星期三) 凌晨2:54
***@***.***&gt;;
抄送:&nbsp;"Ran zhou ***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 支付成功 -- 需要激活码 (Discussion #589)





 
@YayuZhou123
 您好. 这是您专属的激活码:
 QDD8M-KBVDQ-G4FMT-W6YXB-JKRQB

**Comment by YayuZhou123** [comment]
你好，


刚才又试了一下，成功了。


谢谢了。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月19日(星期三) 凌晨2:54
***@***.***&gt;;
抄送:&nbsp;"Ran zhou ***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 支付成功 -- 需要激活码 (Discussion #589)





 
@YayuZhou123
 您好. 这是您专属的激活码:
 QDD8M-KBVDQ-G4FMT-W6YXB-JKRQB

**Reply by MaxMKMax** [reply]
@YayuZhou123 嗯嗯. 那就好. 不客气呢. 谢谢您支持蓝灯.
